ee/ucp/deploy-application-package.md

```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
 
```

Comment:
For Step 5, change the wording of the second sentence to read: "Note that if you select  **Kubernetes Workloads** the **Namespace** drop-down list will display, from which you must select one of the available namespaces.

Reply:
Please put **Orchestrator Mode** and **Namespace** in bold (and be sure that Namespace is capitalized, as it is on-screen).